src/container/streamAnalyticsManager.ts

```diff
@@ -74,11 +119,29 @@ export class StreamAnalyticsManager {
                 }
             }
         } catch (error) {
-            error.message = `Cannot parse Stream Analytics publish job information: ${error.message}`;
+            error.message = `Parse Stream Analytics jobs info failed: ${error.message}`;
             throw error;
         }
     }
 
+    private async getJobSubscription(ASAInfo): Promise<AzureSubscription> {
+        await this.azureAccount.waitForFilters();
```

Comment:
what will happen is not signed in?

Reply:
When we are checking updated, we would check azure login, and if user not login, we would ask him to login first.
![image](https://user-images.githubusercontent.com/5545529/63321297-df252980-c352-11e9-95a1-8bf3f226669c.png)